slate-video-library/src/Pages/Playlistspage/PlaylistViewpage.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { usePlaylistContext } from "../../Components/Context/PlaylistpageContext";
+import Footer from "../../Components/Footer/Footer";
+import Header from "../../Components/Header/Header";
+import Sidebar from "../../Components/Sidebar/Sidebar";
+import SmallVideoCards from "../../Components/SmallVideoCards/SmallVideoCards";
+
+function PlaylistViewpage() {
+  const {
+    setPlaylistId,
+    getVideosFromPlaylists,
+    getVideosFromPlaylistsFn,
+    deleteVideosInsidePlaylistFn,
+  } = usePlaylistContext();
+
+  const params = useParams();
+  setPlaylistId(params);
+  console.log(params);
+
+  useEffect(() => {
+    getVideosFromPlaylistsFn();
+  }, []);
+
+  return (
+    <div>
+      <Header />
+      <Sidebar />
+
+      <div className="watchlater-container">
+        {getVideosFromPlaylists.length <= 0 ? (
```

Comment:
`getVideosFromPlaylists` seems a function name you can use `videosInPlaylist` as variable name